app/assets/stylesheets/application.css.scss

```diff
@@ -129,6 +129,9 @@ section {
   margin-bottom: 100px;
 }
 
+.edit-list {
```

Comment:
Actually, after tinkering, think this might be nonfunctional. Will leave it up to a dedicated frontend team since we're still figuring out the layout.